resources/ctia-default.properties

```diff
@@ -1,10 +1,10 @@
-auth.service.type=allow-all
+ctia.auth.type=allow-all
 ctia.http.port=3000
 ctia.http.min-threads=10
 ctia.http.max-threads=100
 ctia.nrepl.port=3001
 ctia.nrepl.enabled=true
-ctia.store.default=memory
```

Comment:
I think the purpose of having `default` here was to have the ability to define a store per entity, later on you would write something like `ctia.store.actor=memory` and so on.